docs/rtk-query/api/created-api/code-splitting.mdx

```diff
@@ -60,3 +60,24 @@ Any provided tag types or endpoint definitions will be merged into the existing
 Returns an updated and enhanced version of the API slice object, containing the combined endpoint definitions.
 
 This is primarily useful for taking an API slice object that was code-generated from an API schema file like OpenAPI, and adding additional specific hand-written configuration for cache invalidation management on top of the generated endpoint definitions.
+
+For example, `enhanceEndpoints` can be used to modify caching behaviour by changing the values of `providesTags`, `invalidatesTags`, and `keepUnusedDataFor`:
+
+```ts
+const enhancedApi = api.enchanceEndpoints({
```

Comment:
It looks like CI may have failed due to this line:
https://app.netlify.com/sites/redux-starter-kit-docs/deploys/629d1fee7ab8150008c71b33#L301

I will take a closer look and fix this!